tests/acceptance/main-test.js

```diff
@@ -0,0 +1,173 @@
+import { module, test } from 'qunit';
+import { visit, currentURL, click, fillIn, pauseTest, find, findAll } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { currentSession, authenticateSession, invalidateSession } from 'ember-simple-auth/test-support';
+import { tracked } from '@glimmer/tracking';
+import Service from '@ember/service';
+import Adapter from 'ember-local-storage/adapters/local';
+import resetStorages from 'ember-local-storage/test-support/reset-storage';
+import faker from 'faker';
+
+module('Acceptance | Main flow of Application', function(hooks) {
+  setupApplicationTest(hooks);
+  
+  //Preparation before test
+  hooks.beforeEach(function() {
+    //This service is needed to mock the real service that needs a real conection with firebase auth
+    //to retrieve logged user email
+    this.owner.register('service:logged-user', class TestService extends Service {
+      @tracked user = {email: 'user@teste.com'}
+    });
+
+    //This adapter is used to use ember local storage instead of firestore
+    this.owner.register('adapter:application', class TestService extends Adapter {
+   
+    });
+  });
+
+  //This makes the database empty after the end of the test
+  hooks.afterEach(function() {
```

Comment:
you don't need to flush your local storage if your tests are robust enough

Reply:
@tpetrone But if I don´t flush the storage will always get bigger and bigger......